.cbmc-batch/include/proof_helpers/utils.h

```diff
@@ -64,6 +65,16 @@ void assert_array_list_equivalence(
     const struct aws_array_list *const rhs,
     const struct store_byte_from_buffer *const rhs_byte);
 
+/**
+ * Asserts two aws_byte_buf structures are equivalent. Prior to using this function,
+ * it is necessary to select a non-deterministic byte from the rhs aws_byte_buf structure
+ * (use save_byte_from_array function), so it can properly assert all bytes match.
+ */
+void assert_byte_buf_equivalence(
```

Comment:
What's wrong with this function https://github.com/awslabs/aws-c-common/blob/master/.cbmc-batch/source/utils.c#L19

Ahh, I see what this is doing.  Perhaps a clearer comment/name?

Reply:
This is actually form this PR (https://github.com/awslabs/aws-c-common/pull/359), so I'll address this comment there.